teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -27,11 +27,11 @@
 
 public class ValidatorOptions {
 
-  @Mixin(name = "Validator Keys")
-  private ValidatorKeysOptions validatorKeysOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Validator Keys%n")
+  private ValidatorKeysOptions validatorKeysOptions = new ValidatorKeysOptions();
 
-  @Mixin(name = "Validator Proposer")
-  private ValidatorProposerOptions validatorProposerOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Validator Proposer%n")
+  private ValidatorProposerOptions validatorProposerOptions = new ValidatorProposerOptions();
```

Comment:
I think we ultimately want all of these to appear in one "Validator" group.  Removing the `heading` fields from ArgGroup seems to achieve that.